### DISCUSSION TITLE
[Official Course] How to build Web3 Games / Pixel art for Web Games

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#1595] [Ideas]
You could encourage the community of developers, artists, content creators to organize courses for the community.  Something community to community. Mentoring, organization of bootcamps. Multi languages, interaction between students/developers and the community.

Contributing to education and technology also helps to bring new ideas, maintenance and new contributions to the game.

Entrance can be purchased in SFL, shared between the content creators and the game engine. Give a student badge to who join it will give the community the opportunity to evolve into a coder or an artist.

**Comment by adamhannigan** [comment]
@future2back This definitely aligns with our vision of knowledge sharing.

We've been doing weekly pixel art sessions in Discord recently and are looking to move these over to Twitch as the first step.

The actual business/course side of things we will need to leave to the community to take responsibility of - right now we've got to keep our focus on the game development and launching successfully 😅
